seeing tips with perf report requires docs, so notify which package has them
--
diff --git a/tools/perf/builtin-report.c b/tools/perf/builtin-report.c
index 2ee2ecc..0e08d6c 100644
--- a/tools/perf/builtin-report.c
+++ b/tools/perf/builtin-report.c
@@ -651,7 +651,7 @@ static int report__browse_hists(struct report *rep)
 		free(path);
 		path = system_path(DOCDIR);
 		if (perf_tip(&help, path) || help == NULL)
-			help = strdup("Cannot load tips.txt file, please install perf!");
+			help = strdup("Cannot load tips.txt file, install linux-tools-doc to see tips!");
 	}
 	free(path);
 
